pubspec.yaml

```diff
@@ -20,12 +24,19 @@ dependencies:
   hydrated_bloc: ^9.1.5
   intl: ^0.19.0
   json_annotation: ^4.9.0
+  lottie: ^3.1.2
```

Comment:
Why do we need both lottie and rive?

Reply:
We are using Lottie for animations that don't require custom triggers. I prefer to stick with Lottie for displaying simple animations, rather than switching to Rive.